README.md

```diff
@@ -144,8 +144,36 @@ For more information on anchore and anchore-cli, refer to:
 - [Anchore CLI](https://github.com/anchore/anchore-cli)
 - [Circle Orb Registry](https://circleci.com/orbs/registry/orb/anchore/anchore-engine)
 
+## Automated Releases
 
+As part of our CI/CD process, we use a combination of CircleCI, standard-version
+npm package and github-release CircleCI orb to automatically trigger our releases
+and image builds. This process essentially mimics a manual tag and release.
 
-## Additional Notes: 
+On a merge to master, CircleCI is configured to use the mojaloopci github account
+to push the latest generated CHANGELOG and package version number.
 
-- For all put parties callbacks `FSPIOP-Destination` header is considered to be mandatory. 
+Once those changes are pushed, CircleCI will pull the updated master, tag and
+push a release triggering another subsequent build that also publishes a docker image.
+
+### Potential problems
+
+*   There is a case where the merge to master workflow will resolve successfully, triggering
+    a release. Then that tagged release workflow subsequently failing due to the image scan,
+    audit check, vulnerability check or other "live" checks.
```

Comment:
@kleyow @lewisdaly , as much as I'm for this, I think it would be nice to have the issues with image scans and other failures addressed (because failures in scans and these steps as we've been having recently are very frequent), possibly by including them (understanding the overhead involved) in the earlier stages, to avoid failures during release phase

Reply:
Not too clear on your preferred workflow. 

Yes, the current workflow has the scans happening in the merged workflow potentially catching all failures before the release workflow. So if that fails the release workflow won't happen. 

This paragraph is more of outlining that a case of the merged workflow passing scans and the release workflow failing scans is a posibility.

Reply:
@elnyry-sam-k I think we should merge this and create an issue to look/resolve this going forward.

I think its better to have the docs reflect these changes as accurately as possible. Any issues that we have should be addressed via the above created issue.

Reply:
Sure @mdebarros .. thank you